.github/workflows/ci.yml

```diff
@@ -1,29 +1,51 @@
-name: Docker
+name: Tests
 
 on:
-  push:
-    branches:
-      - '*'
-
   pull_request:
+  push:
     branches:
       - master
 
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@m-kuhn To use the container registry I need write rights I believe. May I sollicit write rights?

Reply:
You should have write access for what I can see